web/pages/api/contact.js

```diff
@@ -51,12 +69,25 @@ export default async function handler(request, response) {
   }
 
   // send email
-  const sendEmail = await sendEmailWithSes(contact);
-  if (!sendEmail.success) {
-    return response.status(500).json({
-      success: false,
-      message: "Error sending email",
-    });
+  if (!organization) {
```

Comment:
I'm fine to merge it as is, but it feels bad to me to cue off of `organization` for the action. It seems like a better approach would be to include that in the payload of the form submission. That way we can be more flexible going forward.

Reply:
I don't love this either. I used this because the other form does not have an organization field. 
But do you mean appending the request body with something specific, or would you use something else from the request?

Reply:
Also, should I even be omitting that 'sendEmail' function at all?

Reply:
When it comes to doing a specific action it's usually best to be explicit. Triggering things based on arbitrary form keys will get us in trouble.

Could just be `primaryAction: "sendEmail"` || `primaryAction: "campaignSubscribe"`

You could also send the Mailchimp id in the payload

Reply:
I started this with the list ID for this PR, but set up a todo in basecamp to go back and edit the schema on Sanity for both contact forms with a primaryAction select field.